apps/opencs/view/render/pagedworldspacewidget.cpp

```diff
@@ -127,8 +126,8 @@ void CSVRender::PagedWorldspaceWidget::addVisibilitySelectorButtons (
     CSVWidget::SceneToolToggle2 *tool)
 {
     WorldspaceWidget::addVisibilitySelectorButtons (tool);
-    tool->addButton (Button_Terrain, Mask_Terrain, "Terrain");
-    tool->addButton (Button_Fog, Mask_Fog, "Fog", "", true);
+    tool->addButton (Button_Terrain, SceneUtil::Mask_Terrain, "Terrain");
+    //tool->addButton (Button_Fog, Mask_Fog, "Fog", "", true);
```

Comment:
While fog is not implemented, removing it is different than just moving mask to component level. I wonder what's the idea behind fog here in the first place, still, it's a different issue.